### DISCUSSION TITLE
Option to unblur background image in song select

### DISCUSSION THREAD

**Discussion by kwotaq** [ppy/osu#17392] [Ideas] [answered]
In both the current and upcoming suggested designs the beatmap background image is blurred. Of course this isn't a bad choice however I would like to have the option to reduce/remove the blur since I play with reduced UI scaling and the beatmap image in the top left doesn't suffice. I find recognizing a beatmap by its background in the backdrop a lot more convenient personally!

**Comment by bdach** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/discussions/13687
